Initial introduction and getting started guides for ImageSharp.Drawing and Fonts (#12)
SixLabors/docs: articles/imagesharp.drawing/index.md

# Introduction

>[!WARNING]

> >ImageSharp.Drawing is still considered BETA quality and we still reserve the rights to change the API shapes. WE are yet to priorities performance in our drawing APIs.

Reviewer: ```suggestion
>ImageSharp.Drawing is still considered BETA quality and we still reserve the rights to change the API shapes. We are yet to priorities performance in our drawing APIs.
```

### What is ImageSharp.Drawing?

> ImageSharp.Drawing is a library build on top on ImageSharp to providing 2D Drawing extensions on top of ImageSharp.

Reviewer: ```suggestion
ImageSharp.Drawing is a library build on top on ImageSharp to providing 2D Drawing extensions.
```

> ImageSharp.Drawing is designed from the ground up to be flexible and extensible. The library provides API endpoints for common vector and text processing operations adds the building blocks for building custom image.

Reviewer: ```suggestion
ImageSharp.Drawing is designed from the ground up to be flexible and extensible. The library provides API endpoints for common vector and text processing operations adding the building blocks for building custom images.
```
